platform-hub-api/app/models/concerns/read_only_associations.rb

```diff
@@ -0,0 +1,40 @@
+module ReadOnlyAssociations
+  extend ActiveSupport::Concern
+
+  class_methods do
+
+    # Example
+    # read_only_associations :foo, :bar, {baz: :polymorphic}
+    def read_only_associations *associations
```

Comment:
Is this actually used anywhere?

Reply:
Also, I don't think you need this – sounds you can just do `readonly: true` on the association declarations in the model.

Reply:
It's not used anywhere, accidental commit.